### PR TITLE
added textures

### PR DESCRIPTION
**Pull request by JamesCoyne**
textures!

### CODE REVIEW

File: src/components/Space/GenericSpace.tsx [hunk 1]
```diff
@@ -12,46 +12,55 @@ interface GenericSpace {
...
+//     );
+//   }
+// =======
+const GenericSpace: React.FC<GenericSpace> = props => {
+  return(
+    <Suspense fallback="none">
+      {props.type === 'empty' ? <Empty visited={props.visited} path={props.path} /> : <Wall type={props.type} />}
+    </Suspense>
+  );
+// >>>>>>> Stashed changes
```
Comment (ShawnToubeau) [inline src/components/Space/GenericSpace.tsx:63]: This looks like a merge conflict, is this intended?

Comment (JamesCoyne) [reply src/components/Space/GenericSpace.tsx:63]: Yes, sorry, this is mike's code for coloring closed sets and open sets. I should have made an inline comment about it - It's intentionally left in for future reference if and when we texture open and closed sets.

Comment (ShawnToubeau) [reply src/components/Space/GenericSpace.tsx:63]: Ah ok cool

Comment (uptonm) [reply src/components/Space/GenericSpace.tsx:63]: ye, lookin good :)

**Pull request merged**